assets/loc/translations.csv

```diff
@@ -557,6 +575,7 @@ OSHI_BP1_005_FULL_TEXT,When you use a Red Holomem's Ability/Arts that Archives c
 OSHI_BP1_006_FULL_TEXT,When your Red Holomem is downed on the opponent's turn: lose 1 less LIFE. You may take it and all stacked Holomem into your hand.,"상대의 턴에 자신의 적색 홀로멤이 다운했을 때 사용할 수 있다: 자신의 줄어들 라이프 -1. 다시, 다운한 1명을 골라 그 홀로멤을 포함해 겹쳐져 있던 홀로멤 전부를 패로 되돌린다.",あなたの赤いホロメンが対戦相手のターン中にダウンした時: 1のライフを失う。そのホロメンとすべてのスタックされたホロメンを手札に加えることができる。,Cuando uno de tus Holomems Rojos es Derrotado en el turno oponente: pierde 1 menos de VIDA. Puedes tomarlo y todas sus cartas apiladas a tu mano.
 OSHI_BP1_007_FULL_TEXT,When this or a Blue Holomem deals damage to your opponent's Back Holomem: deal 50 damage to that Holomem,이 오시 홀로멤이나 자신의 청색 홀로멤이 상대의 백 홀로멤에게 대미지를 주었을 때 사용할 수 있다: 그 상대의 백 홀로멤 1명에게 특수 대미지 50을 준다.,このホロメンまたは青いホロメンが対戦相手のバックホロメンにダメージを与えた時: そのホロメンに50のダメージを与える。,Cuando esta o un Holomem Azul hacen daño a un holomem Trasero oponente: haz 50 daño a ese Holomem.
 OSHI_BP1_007_2_FULL_TEXT,When your Blue Holomem deals damage to your opponent's Center or Collab Holomem: deal equal special damage to one of your opponent's Back Holomem,자신의 청색 홀로멤이 상대의 센터 홀로멤이나 콜라보 홀로멤에게 대미지를 주었을 때 사용할 수 있다: 상대의 백 홀로멤 1명에게 그것과 같은 수치의 특수 대미지를 준다.,あなたの青いホロメンが対戦相手のセンターまたはコラボホロメンにダメージを与えた時: 対戦相手のバックホロメンの1つに等しい特別ダメージを与える。,"Lanza un dado. Si el resultado es 3 o menos, Archiva uno de los Aplausos de los holomems del oponente.\nSi tu Oshi es Nanashi Mumei, puedes activar el siguiente efecto en vez: (Una vez por juego) Archiva 2 Aplausos del Centro oponente."
+AYAME_09_ART_FULL_TEXT,Archive 0-3 cards from your hand. Deal 40 special damage per to your opponent's Center.,,,
```

Comment:
I'd like to make sure we have translations, because if it's blank i assume it falls back to the tag which will not be great.
Maybe just use chatgpt for now and make a list somewhere of translations to check for people who want to help us out.